src/screens/InboundReceiveDetail/index.tsx

```diff
@@ -292,6 +303,21 @@ const InboundReceiveDetail = () => {
           label={'Lot Number'}
           onChange={onChangeLotNumber}
         />
+        <SelectDropdown
+            data={['Select lot status','APPROVED', 'RECALLED', 'ON_HOLD', 'QUARANTINED', 'EXPIRED', 'RESERVED', 'DAMAGED']}
```

Comment:
'Select lot status' should not be an option on the data list. It should be a placeholder.